libs/ui-components/src/components/Fleet/CreateFleet/steps/UpdatePolicyStep.tsx

```diff
@@ -140,6 +160,23 @@ const UpdatePolicyStep = () => {
               }
             />
             {disruptionBudget.isAdvanced && <UpdateStepDisruptionBudget />}
+
+            {/* Update (and download) policies */}
+            <Checkbox
+              label={
+                <WithHelperText
+                  showLabel
+                  ariaLabel={t('Set update policies')}
+                  content={t('Update policies allow you to control when updates should be downloaded and applied.')}
+                />
+              }
+              isChecked={updatePolicy.isAdvanced}
+              id="advUpdatePolicy"
+              onChange={(_ev: React.FormEvent<HTMLInputElement>, toAdvanced: boolean) =>
+                onSettingsChange('update-policies', toAdvanced)
+              }
+            />
+            {updatePolicy.isAdvanced && <UpdateStepUpdatePolicy />}
```

Comment:
should the children be a checkbox body instead ? Could we get rid of custom styling then ?
```suggestion
            {/* Update (and download) policies */}
            <Checkbox
              label={
                <WithHelperText
                  showLabel
                  ariaLabel={t('Set update policies')}
                  content={t('Update policies allow you to control when updates should be downloaded and applied.')}
                />
              }
              isChecked={updatePolicy.isAdvanced}
              id="advUpdatePolicy"
              onChange={(_ev: React.FormEvent<HTMLInputElement>, toAdvanced: boolean) =>
                onSettingsChange('update-policies', toAdvanced)
              }
              body={updatePolicy.isAdvanced && <UpdateStepUpdatePolicy />}
            />
```

Reply:
Good idea, done!